sw/device/tests/dif/dif_kmac_smoketest.c

```diff
@@ -155,6 +155,80 @@ const shake_test_t shake_tests[] = {
     },
 };
 
+#define DIGEST_LEN_CSHAKE_MAX 4
+
+/**
+ * cSHAKE test description.
+ */
+typedef struct cshake_test {
```

Comment:
nit: This file is covering many tests. Consider splitting this into `dif_kmac_cshake_test.c`, `dif_kmac_sha3_test.c` etc.

Reply:
Yeah I think that's a good idea. I've put the cSHAKE tests into a separate file and will split up the other tests in a separate PR.